source/backend/api/Models/Concepts/CompensationRequisition/CompensationRequisitionMap.cs

```diff
@@ -23,11 +23,9 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.ExpropriationNoticeServedDate, src => src.ExpropNoticeServedDt)
                 .Map(dest => dest.ExpropriationVestingDate, src => src.ExpropVestingDt)
                 .Map(dest => dest.SpecialInstruction, src => src.SpecialInstruction)
-                .Map(dest => dest.Payees, src => src.PimsAcquisitionPayees)
```

Comment:
It's okay to remove the compensation payee mapping here but I was expecting to see the payee fields (that were moved into compensation requisition) mapped here as well . For example:

- AcquisitionOwnerId
- InterestHolderId
- IsPaymentInTrust
- etc

Reply:
Same as above, this PR is about getting the scaffold updated and the backend compiling. Another PR will need to take a holistic approach to the compensation requisition functionality alignment.